src/pages/waiting-room-to-car/waiting-room-to-car.ts

```diff
@@ -178,14 +179,17 @@ export class WaitingRoomToCarPage extends BasePageComponent {
         map(isManual),
       ),
       tellMeQuestionSelected$: currentTest$.pipe(
+        select(getTestData),
```

Comment:
Same as on Office, we're doing `currentTest$.pipe`, wouldn't have thought that `getTestData` was necessary.